README.md

```diff
@@ -1,64 +1,148 @@
-# Biconomy Smart Account (Smart Contract Wallet) Overview
+![Solidity](https://img.shields.io/badge/Solidity-0.8.17-blue.svg) ![Hardhat](https://img.shields.io/badge/Framework-Hardhat-brightgreen.svg) ![Foundry](https://img.shields.io/badge/Framework-Foundry-orange.svg) ![Test Coverage](https://img.shields.io/badge/Coverage-45%25-red.svg)
 
-Biconomy Modular Smart Account is an [EIP-4337](https://eips.ethereum.org/EIPS/eip-4337) compatible modular smart contract wallet.
-Smart Account is ownerless by nature. UserOp and txns validation happens in Authorization Modules.
+# Biconomy Smart Account: Leading Implementation of Account Abstraction 🌐
 
-Smart Account is designed in such a way that it is:
+Biconomy Smart Account is a smart contract wallet focused on implementing Account Abstraction. It builds on the core concepts of Gnosis and Argent safes and is compliant with [EIP-4337](https://eips.ethereum.org/EIPS/eip-4337) and [EIP-6900](https://eips.ethereum.org/EIPS/eip-6900).
 
-- Modular => highly customizable and extandable. 
-- Cheap to deploy proxy copies of an implementation (user wallets)
-- Wallet addresses are counterfactual in nature (you can know the address in advance and users can have the same address across all EVM chains)
-- Deployment cost can be sponsored 
+![Biconomy Account Abstraction Banner](./assets/readme/biconomy-account-abstraction.png)
 
-# How to run the project
+## 📜 Smart Contracts
+
+- **BaseSmartAccount.sol**: An abstract contract implementing the EIP4337 IWallet interface.
```

Comment:
We should use the word **ERC** for 4337 instead of **EIP**  IF you check the proposal they are also using the word ERC